src/handlers/testing/exec/util/run-manager.ts

```diff
@@ -408,58 +408,56 @@ export class RunManager {
     const runId = this.currentRunId({
       testExternalId: args.testExternalId,
     });
-    const payload = {
-      testCaseHash: args.testCaseHash,
-      testCaseBody: args.testCaseBody,
-      testCaseOutput: args.testCaseOutput,
-      testCaseEvents: events,
-      testCaseDurationMs: args.testCaseDurationMs,
-      testCaseRevisionUsage: args.testCaseRevisionUsage,
-      testCaseHumanReviewInputFields: args.testCaseHumanReviewInputFields,
-      testCaseHumanReviewOutputFields: args.testCaseHumanReviewOutputFields,
-    } as const;
-
-    let resultId: string;
-    try {
-      const { id } = await this.post<{ id: string }>(
-        `/runs/${runId}/results`,
-        payload,
-      );
-      resultId = id;
-    } catch (err) {
-      try {
-        const parsedError = zHttpError.parse(
-          JSON.parse((err as Error).message),
-        );
-        if (parsedError.status === 413) {
-          // If the /results request fails with a content too large error, retry with an empty output
-          const { id } = await this.post<{ id: string }>(
-            `/runs/${runId}/results`,
-            {
-              ...payload,
-              testCaseOutput: '',
-            },
-          );
-          resultId = id;
-        } else {
-          throw err;
-        }
-      } catch {
-        throw err;
-      }
-    }
+
+    const { id: resultId } = await this.post<{ id: string }>(
+      `/runs/${runId}/results`,
+      {
+        testCaseHash: args.testCaseHash,
+        testCaseDurationMs: args.testCaseDurationMs,
+        testCaseRevisionUsage: args.testCaseRevisionUsage,
+      },
+    );
 
     if (!this.testCaseHashToResultId[args.testExternalId]) {
       this.testCaseHashToResultId[args.testExternalId] = {};
     }
     this.testCaseHashToResultId[args.testExternalId][args.testCaseHash] =
       resultId;
 
-    await this.runUIBasedEvaluators({
-      testExternalId: args.testExternalId,
-      testCaseId: resultId,
-      testCaseHash: args.testCaseHash,
-    });
+    // We still want to try the other endpoints if 1 fails
+    await Promise.allSettled([
```

Comment:
you could assign this to a variable and log the errors for the ones that failed. to log to the CLI output you can do:

```ts
emitter.emit(EventName.CONSOLE_LOG, {
  ctx: 'cli',
  level: 'warn',
  message: '...',
});
```

Reply:
ya was debating that, went ahead and added some. let me know what you think of these.